README.md

```diff
@@ -234,9 +234,11 @@ secret, err := client.Secrets.KvV2Read(
 )
 ```
 
-#### Adding Custom Headers and Custom Query Parameters
+#### Adding Custom Headers and Query Parameters
```

Comment:
there's potentially already query parameters correct?
if so, would maybe make that clear that they are conditional and not replacing ones already included in the request

Reply:
Good point, let me try to reword this